drivers/video/Kconfig

```diff
@@ -31,6 +31,21 @@ config VIDEO_BUFFER_POOL_ALIGN
 	int "Alignment of the video pool’s buffer"
 	default 64
 
+config VIDEO_BUFFER_USE_SHARED_MULTI_HEAP
+	bool "Use shared multi heap for video buffer"
```

Comment:
I think this option would depend on the presence of an external memory device,  I'm wondering if would be good to cheate a `chosen` property on the device tree to indicate that presence and use it to drive this option.

for example:

```
{
  chosen, zephyr-ext-ram = &ext-ram0;
}

```

Reply:
I'm not sure. #76980 extends SMH attributes to include external memory as a possible heap capability. This feature provides more capabilities that don't require an external memory.

Reply:
That makes sense.